loaders/src/glTF/2.0/glTFLoader.ts

```diff
@@ -116,6 +116,7 @@ export class GLTFLoader implements IGLTFLoader {
     private _babylonScene: Scene;
     private _rootBabylonMesh: Mesh;
     private _defaultBabylonMaterialData: { [drawMode: number]: Material } = {};
+    private static _buffersToDispose = new Array<VertexBuffer>();
```

Comment:
```suggestion
    private static readonly _buffersToDispose = new Array<VertexBuffer>();
```

Reply:
Is it possible to push item into a readonly array?

Reply:
yes

Reply:
Yes the `readonly` here just means the `_buffersToDispose` variable cannot be reassigned. That's different from the array itself being readonly (`ReadonlyArray<VertexBuffer>` or `Readonly<Array<VertexBuffer>>` or `Readonly<VertexBuffer[]>` or `readonly VertexBuffer[]` 😬), in which case items cannot be pushed.